platform-dom/src/main/resources/com/softicar/platform/dom/elements/dom-day-input-style.css

```diff
@@ -1,4 +1,5 @@
 
 .DomDayInput {
 	display: inline-grid;
-}
\ No newline at end of file
+	grid-auto-columns: auto max-content;
```

Comment:
unrelated fix; the text input part of `DomDayInput` previously was not stretched horizontally, which looked poor in EMF input forms when displayed below regular text inputs